src/backend/apps/clusters/connector.py

```
@@ -299,11 +299,17 @@ def sync_common(self, sync_type):
                         )
                 response_data.append(result["id"])
         if sync_type == 'job_template':
+            skipped_count = 0
             for key, value in db_data.items():
```

Ruff (0.15.13)

[warning] Loop control variable `key` not used within loop body

Rename unused `key` to `_key`

(B007)